### DISCUSSION TITLE
"Show windows side by side" as a separate component?

### DISCUSSION THREAD

**Discussion by 8979867239498** [valinet/ExplorerPatcher#3048] [General]
This is a great tool, brings back the best feature called "Show windows side by side" that Microsoft shockingly and disgracefully removed from windows 11 - however ep_setup really changes some of the non-disgraceful Windows 11 taskbar behaviors that are actually good... Had to uninstall for that reason. Is there a way to install ep_setup, but modify the installer to just get the "show windows side by side" feature back and none of the other changes or bells and whistles?

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Can you list the settings that you say are non-disgraceful in the windows 11 taskbar that EP changed?

Please include any other software you have installed that modifies the UI of windows.

**Comment by Amrsatrio** [comment]
> changes some of the non-disgraceful Windows 11 taskbar behaviors that are actually good

Please explain which behavior; usually such modifications to the vanilla Windows 11 taskbar can be disabled. We have no plans of making a smaller EP build that consists of subcomponents EP offers.

**Reply by 8979867239498** [reply]
Is there any way to extract the code for "Show windows side by side"? Don't need a full build. Could simply be a standalone script...  cmd, ps1, vbs, or any other executable script to evenly tile all open windows. This sounds so simple, but the behavior isn't matched by any other app, it's removed from windows 11, and no one has released a viable workaround for years.
